### PR TITLE
Adds a charcoal pen (port)

### PR DESCRIPTION
**Pull request by boodaliboo**
Original PR: https://github.com/tgstation/tgstation/pull/48219
"It simply adds a craftable pen made with a single piece of wood and some ash. It writes in a unique font with a dim grey color."
#### Changelog

:cl:  
rscadd: Added a new charcoal pen created with ash and wood
/:cl:

### CODE REVIEW

File: code/__DEFINES/misc.dm [hunk 1]
```diff
@@ -88,6 +88,7 @@
 #define GAME_STATE_SETTING_UP	2
 #define GAME_STATE_PLAYING		3
 #define GAME_STATE_FINISHED		4
+#define CHARCOAL_FONT "Candara"
```
Comment (Bibby0110) [inline code/__DEFINES/misc.dm]: Define this in the fonts section...

Comment (boodaliboo) [reply code/__DEFINES/misc.dm]: oh lol

**Comment by azzzertyy** [comment]
make it able to be eat to ingest charcoal in the bloodstream because it makes sense

**Comment by boodaliboo** [comment]
> make it able to be eat to ingest charcoal in the bloodstream because it makes sense

no only activated charcoal works like that which is ss13 one

**Review by Partheo — APPROVED** [review]

**Comment by azzzertyy** [comment]
i believe activated charcoal is only charcoal in liquid form, as well as slightly less dilute and concentrated, id say itd fit if it could act as this, but id understand if it didnt

**Review by JamieD1 — APPROVED** [review]

**Pull request merged**